### PR TITLE
Adds Leemabeens

### PR DESCRIPTION
**Pull request by sqnztb**
## About The Pull Request

Per feedback from jolly in #55

## Why It's Good For The Game

It's not.

## Changelog
:cl:
add: leemabeens
/:cl:

### CODE REVIEW

**Review by Jolly-66 — COMMENTED** [review]
donde estas la foto?

**Comment by Jolly-66** [comment]
disgusting you made them a subparent of soybeans

this is not kino soft soyboy moment

**Comment by sqnztb** [comment]
just borrowing the sprite.

i'm a m*pper/c*der not a spriter.

**Comment by Jolly-66** [comment]
whenever melbert squash and merges the upstream bs i'll make you lima sprites

File: code/modules/vending/megaseed.dm [hunk 1]
```diff
@@ -43,7 +43,8 @@
 					/obj/item/seeds/tower = 3,
 					/obj/item/seeds/watermelon = 3,
 					/obj/item/seeds/wheat = 3,
-					/obj/item/seeds/whitebeet = 3)
+					/obj/item/seeds/whitebeet = 3,
+					/obj/item/seeds/leemabeens = 3)
```
Comment (MrMelbert) [inline code/modules/vending/megaseed.dm:47]: comment

Comment (Jolly-66) [reply code/modules/vending/megaseed.dm:47]: can't we make a child of it in a modular file that just gets the other components?

File: jollystation_modules/code/modules/hydroponics/grown/beans.dm [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+/obj/item/food/grown/leemabeens
+	seed = /obj/item/seeds/leemabeens
+	name = "leema beens"
+	desc = "What exactly are these supposed to be, anyway?"
+	gender = PLURAL
+	icon = 'icons/obj/clothing/masks.dmi'
+	icon_state = "gas_alt"
+	bite_consumption_mod = 2
+	foodtypes = VEGETABLES
+	tastes = list("mapper frustration" = 1)
```
Comment (Jolly-66) [inline jollystation_modules/code/modules/hydroponics/grown/beans.dm:28]: what the fuck

File: code/modules/vending/megaseed.dm (hunk 1 above)
Comment (MrMelbert) [reply code/modules/vending/megaseed.dm:47]: there's an already existing implementation of modular vending machines that just adds it to the list via its own file i just need to port it

**Comment by Jolly-66** [comment]
oh yeah im gonna add sprites to this suffer at my inability to maker lima beans

**Comment by Jolly-66** [comment]
should i wait until melbert ports over the modular vendors

**Comment by Jolly-66** [comment]
taking of DNM but its depending if we wanna wait until melbert ports over custom vendors

**Comment by MrMelbert** [comment]
image
I just realized you branched off this fork instead of your own, pretty based

also #63

**Comment by Jolly-66** [comment]
@sqnztb #63

**Comment by Jolly-66** [comment]
we'll make this more legit later

**Pull request merged**